src/main/java/views/TeamEditorPane.java

```diff
@@ -0,0 +1,4 @@
+package views;
+
+public class TeamEditorPane {
+}
```

Comment:
Overall, good clear code, clear style too. Some **minor** inconsistenties in the style here and there, which were already pointed out by the other reviewers. 
Furthermore, the naming of the functions and variables could be done a little better, most of them were good and clear, however, as pointed out, some names should be more self explanatory. 
I advise adding more comments, I've seen a few which were sufficient, but could be improved upon. Some sections do not have any comments at all, which is not very good practise. Over explaning code with comments is never a bad idea! 
All in all, this code is nice. Good job, and I hope I helped. Good luck with assignment 3!